1-exercises/D-array-filter/exercise.js

```diff
@@ -4,11 +4,17 @@
   It is decided that array items which are not pairs should be filtered out
   - Finish the statement on line 11 to produce an array with valid content
   - Do not edit any of the existing code
+  ------------Вам предоставляется программа, которая регистрирует пары между наставниками и учениками.
```

Comment:
Again, be careful with what you commit before you commit it. Using `git diff` before committing should help making sure you're committing only what's necessary